Formula/htmltest.rb

```diff
@@ -0,0 +1,29 @@
+class Htmltest < Formula
+  desc "HTML validator written in Go"
+  homepage "https://github.com/wjdp/htmltest"
+  url "https://github.com/wjdp/htmltest/archive/v0.14.0.tar.gz"
+  sha256 "add922cf1dd957afba2927d401184c1d2331983a6d8ed96dd10f5001930cebf8"
+  license "MIT"
+
+  depends_on "go" => :build
+
+  def install
+    system "go", "build", *std_go_args
+  end
+
+  test do
+    (testpath/"test.html").write <<~EOS
+      <!DOCTYPE html>
+        <html>
+          <body>
+            <nav>
+            </nav>
+            <article>
+              <p>Some text</p>
+            </article>
+          </body>
+        </html>
+    EOS
+    system "#{bin}/htmltest", "test.html"
```

Comment:
Thanks, @jloh. Does this spit out any output? We should probably test that with an `assert_match` or `assert_equal`.

Reply:
It doesn't, if everything passes there is no output:

```
❯ brew test htmltest
==> Testing htmltest
==> /usr/local/Cellar/htmltest/0.14.0/bin/htmltest test.html
```

Reply:
Does `htmltest test.html` also show nothing if you run it in your shell, outside of brew?

I guess I'm just a little wary of tests that do nothing since we don't know when they fail. Perhaps it may be better to put broken HTML in `test.html` and check for an expected error message?

Reply:
Ahh apologies, I assumed `brew test` showed stdout but it didn't. I've added an `assert_match` because it outputs the time it was started at:

```
❯ htmltest test.html
htmltest started at 02:28:26 on .
========================================================================
✔✔✔ passed in 737.071µs
```

Thankyou for your help!

Reply:
Awesome, this is really helpful. Thanks for working on the test. 

A lot of contributors find it a pain, but the test is really essential in our CI to make sure things work. We have a lot of formulae with not very good tests that actually still pass even though the formula is broken.

Reply:
Completely understand! Thanks for your help. First PR for a completely new formula.